prime-router/metadata/hl7_mapping/ORU_R01/base/observation-result.yml

```diff
@@ -0,0 +1,125 @@
+name: ORU-R01-observation-result
+constants:
+  hl7OBXField: /PATIENT_RESULT/ORDER_OBSERVATION(%{orderIndex})/OBSERVATION(%{resultIndex})/OBX
+elements:
+  - name: result-set-ID
+    value: [ '%resultIndex + 1' ]
+    hl7Spec: [ '%{hl7OBXField}-1' ]
+
+  - name: result-value-type-st
+    value: [ '''ST''' ]
+    hl7Spec: [ '%{hl7OBXField}-2' ]
+
+  - name: result-value-type-cwe
+    condition: '%resource.value.exists()'
+    value: [ '''CWE''' ]
+    hl7Spec: [ '%{hl7OBXField}-2' ]
+
+  - name: observation-identifier-code
+    value: [ '%resource.code.coding.code' ]
+    hl7Spec: [ '%{hl7OBXField}-3-1' ]
+
+  - name: observation-identifier-text
+    value: [ '%resource.code.text' ]
+    hl7Spec: [ '%{hl7OBXField}-3-2' ]
+
+  - name: observation-identifier-coding-system
+    value: [ '%resource.code.coding.version' ]
+    hl7Spec: [ '%{hl7OBXField}-3-3' ]
+
+  # override to handle LN translated to loinc L on HL7->FHIR conversion
+  - name: observation-identifier-coding-system-ln
+    resource: '%resource.code.coding'
+    condition: '%resource.system = "http://loinc.org" and %resource.version = "L"'
+    value: [ '''LN''' ]
+    hl7Spec: [ '%{hl7OBXField}-3-3' ]
+
+  - name: observation-identifier-alternate-code
+    value: [ '%resource.code.extension(%`rsext-alternate-identifier`).value.coding.code' ]
+    hl7Spec: [ '%{hl7OBXField}-3-4' ]
+
+  - name: observation-identifier-alternate-text
+    value: [ '%resource.code.extension(%`rsext-alternate-identifier`).value.coding.display' ]
+    hl7Spec: [ '%{hl7OBXField}-3-5' ]
+
+  - name: observation-identifier-alternate-system
+    value: [ '%resource.code.extension(%`rsext-alternate-identifier`).value.coding.system' ]
+    hl7Spec: [ '%{hl7OBXField}-3-6' ]
+
+  - name: observation-value-sub-id
+    value: [ '%resource.extension(%`rsext-sub-id`).value' ]
+    hl7Spec: [ '%{hl7OBXField}-4' ]
+
+  - name: observation-value-st
+    condition: '%resource.value is string'
+    value: [ '%resource.value' ]
+    hl7Spec: [ '%{hl7OBXField}-5' ]
+
+  - name: observation-value-cwe-identifier
+    condition: '%resource.value.coding.exists()'
+    value: [ '%resource.value.coding.code' ]
+    hl7Spec: [ '%{hl7OBXField}-5-1' ]
+
+  - name: observation-value-cwe-identifier
+    condition: '%resource.value.exists()'
```

Comment:
Change to `%resource.value is CodeableConcept`

Reply:
fixed